scripts/Dockerfile

```diff
@@ -8,7 +11,22 @@ ADD . $MT_SRC_DIR
 RUN $MT_SRC_DIR/scripts/build.sh
 RUN cp -r $MT_SRC_DIR/build /tmp/build
 
+####################
+# mt-gateway image #
+####################
+FROM alpine:3.10.0 AS mt-gateway
 
+RUN mkdir -p /etc/gw
+COPY scripts/config/storage-schemas.conf /etc/gw/storage-schemas.conf
```

Comment:
`/etc/gw` feels kind of gross, but that's the current default...

Possible alternatives:
* `/etc/gateway`
* `/etc/mt-gateway`
* `/etc/metrictank-gateway`
* `/etc/metrictank`

Reply:
does mt-gateway take a similar config file as tsdb-gw? are its config flags a subset of tsdbgw's?
anyway i suggest we use the established `/etc/metrictank/` directory for an mt-gateway.ini file.

Reply:
>does mt-gateway take a similar config file as tsdb-gw? 

Not at the moment

>are its config flags a subset of tsdbgw's?

I believe so

>anyway i suggest we use the established /etc/metrictank/ directory for an mt-gateway.ini file.

I'll toss `storage-schemas.conf there as well.  Might do the ini file as a follow-up PR

Reply:
we already have /etc/metrictank/storage-schemas.conf 
we should be able to use the same file for MT and mt-gateway, at least by default

Reply:
cool, that's what I was planning on

Reply:
Updated in 55430f640cca986d5fb475c7769a9470a7b98886 after rebasing post #1629